src/commands.c

```diff
@@ -7311,6 +7328,184 @@ void SetMidairMinHeight()
 }
 
 void W_SetCurrentAmmo();
+
+void ToggleFreshTeams()
+{
+	int k_freshteams = bound(0, cvar("k_freshteams"), 2);
+
+	if (match_in_progress)
```

Comment:
remove match_in_progressm in new commands, it is redundant, is_rules_change_allowed() checks it as well but also with user warnings.

Reply:
removed.